designs/Verify-Verbosity.md

```diff
@@ -0,0 +1,416 @@
+# `dotnet nuget verify --verbosity`
+
+* Status: **In Review**
+* Author: [Kartheek Penagamuri](https://github.com/kartheekp-ms)
+* Issue: [#10316](https://github.com/NuGet/Home/issues/10316) - dotnet nuget verify is too quiet
+
+## Problem background
+
+`dotnet nuget verify` command doesn't provide any output by default due to logical error in the implementation. If the user didn't pass value for `--verbosity` option, then the log level is set to [`LogLevel.Minimal`](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L58). Customers need information about packages’ signatures to [manage package trust boundaries](https://docs.microsoft.com/en-us/nuget/consume-packages/installing-signed-packages).
+
+## Who are the customers
+
+Package consumers that use the `dotnet nuget verify` command to verify package signature and configure NuGet security policy.
+
+## Goals
+
+* Change the output of `dotnet nuget verify` command for various `--verbosity` options.
+
+## Non-goals
+
+* Add `--verbosity` option to `dotnet nuget verify` command.
+
+## Solution
+
+The details that should be displayed on each verbosity level are described below. Each level should display the same as the level below plus whatever is specified in that level. In that sense, `quiet` will be give the least amount of information, while `diagnostic` the most.
+
+​                                  | `q[uiet]` | `m[inimal]` | `n[ormal]` | `d[etailed]` | `diag[nostic]`
```

Comment:
Great table, thank you so much for providing it!

The big question on this spec is..."What is considered the minimal amount of input for users to acknowledge verification?". I think @dtivel already have a proposal here which best matches the expectations:

https://github.com/NuGet/NuGet.Client/pull/3774#issuecomment-740216665

Reply:
I followed Damon's proposal in the comment to come up the log messages displayed for `minimal` verbosity.

Reply:
I added few examples at the end in the spec with output for various verbosity levels.